robot-server/robot_server/sessions/router/actions_router.py

```diff
@@ -0,0 +1,89 @@
+"""Router for /sessions endpoints."""
```

Comment:
Maybe something like 
```suggestion
"""Router for /sessions actions endpoints."""
```
for clarity?